components/brave_rewards/resources/ui/components/panelWelcome/index.tsx

```diff
@@ -75,6 +77,10 @@ export default class PanelWelcome extends React.PureComponent<Props, {}> {
       }
     }
 
+    const descriptionLocale = onlyAnonWallet && this.locale.desc === 'welcomeDescTwo'
```

Comment:
maybe we should do this in `locale` function?

Reply:
moved